.github/workflows/release.yml

```diff
@@ -46,14 +47,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # - target: x86_64-unknown-linux-musl
-          #   os: ubuntu-latest
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
```

Comment:
This needs to be `-gnu` now, right?

Reply:
I actually didn't want to comment this out, I'll fix all that up in @andreaphylum's PR.